src/styles.css

```diff
@@ -63,6 +63,11 @@ pre.shiki .language-id {
   display: none;
 }
 
+/** Hide twoslash includes' definition code-blocks. */
+.language-twoslash {
+  display: none;
+}
```

Comment:
Good idea! Would be nice to strip these code blocks entirely from the markup, but this is fine for the meantime. Want to jot a `TODO` here?

Reply:
Thanks for your feedback! I've tried to get rid of the whole block, but: 
1. Markdown-it's highlight function [has to return a string](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/markdown-it/lib/index.d.ts#L79-L86). Intuitively , I would have expected the ability to skip the block by returning some falsy value, but nope!
2. an empty string actually means "print without transforming anything", so this would not work either.

Unfortunately I did not have the opportunity to dig it further yet, so I ended up adding the `TODO` as you suggested! 

I've still included a modification based on [this note](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/6d0c4ca54f9b5aee29bb21c9b85be0768979b417/types/markdown-it/lib/index.d.ts#L83) so that the hidden `.language-twoslash`'s `pre` at least remains empty (instead of having the whole content in the final DOM). Not yet perfect, but it should still be better:

```hml
<div class="language-twoslash">
  <button title="Copy Code" class="copy"></button>
  <span class="lang">twoslash</span>
  <pre></pre> <!-- always empty -->
</div>
```

Reply:
Thanks for the info! Fine for now and let's ship this thing.